README.md

```diff
@@ -119,10 +111,48 @@ to be in the PATH. This can be achieved with:
 $ export PATH=/opt/intel/oneapi/compiler/latest/linux/bin-llvm:$PATH
 ```
 
+#### Using the docker image
+
+A docker image is available and provides an up-to-date, one-command install environment. You can either build it from the [Dockerfile](https://github.com/soda-inria/sklearn-numba-dpex/blob/main/docker/Dockerfile) :
```

Comment:
I think using a relative is preferable to local interaction and should works properly on GitHub interfaces.
```suggestion
A docker image is available and provides an up-to-date, one-command install environment. You can either build it from the [Dockerfile](./docker/Dockerfile) :
```